candidate-standard/openapi/schemas/collection.yaml

```diff
@@ -58,24 +58,23 @@ properties:
   extent:
     $ref: extent.yaml
   crs:
-    description: list of the coordinate reference systems the collection results can
-      be produced in
+    description: list of the coordinate reference systems the collection results can support
     type: array
     items:
-      $ref: crsObject.yaml
-  referencemetadata:
-    description: Property to contain any extra metadata information that is specific
-      to the collection
-    type: object
-  distanceunits:
-    description: list of distance units radius distance values can be specified in
-    type: array
-    items: 
       type: string
-    example:
-      - km
-      - miles
-  outputformat:
+  data-queries:
```

Comment:
This is not a valid OpenAPI. I believe it should be
```
  data-queries:
    description: Detailed information relevant to individual query types
    type: object
    properties:
      position:
        $ref: dataQuery.yaml
      radius:
        $ref: dataQuery.yaml
      area:
        $ref: dataQuery.yaml
      cube:
        $ref: dataQuery.yaml
      trajectory:
        $ref: dataQuery.yaml
      corridor:
        $ref: dataQuery.yaml
 ```

Reply:
Thanks @iandruska-ibl, I have uploaded now upload a correction